index.js

```diff
@@ -2,17 +2,72 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+//We needed to add in the "defer" into the script tag so the DOM would run after the hmtl creates the #header part.
 
 /***** Deliverable 2 *****/
-
+header.style.color = "crimson" ;
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+//  console.log('This is what the traveler object looks like: ', traveler)
+// On line 8 in index.html we run   <script src="data.js"></script> with let's us use anything in that file. 
+
+let profile = document.getElementById("profile")
+//console.log(profile)
+
+let img = profile.querySelector('img');
+let name = profile.querySelector('h2');
+let nickname = profile.querySelector('em');
+
+// console.log(nickname)
+// console.log(name)
+
+img.src = traveler.photo;
+img.alt = traveler.name;
+name.textContent = traveler.name;
+nickname.textContent = traveler.nickname;
 
+/* <div id="profile">
+<img src="https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg"
+  alt="{traveler name}">
+<h2>{traveler Name}</h2>
+<em>{traveler Nickname}</em>
+</div> */
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function (sightingsObj) {
 
+let animals = document.querySelector("ul#animals")
+//console.log(animals)
+let li = document.createElement('li')
+let p = document.createElement('p')
+let img = document.createElement('img')
+let a = document.createElement('a')
+
+li.setAttribute('data-id', `${sightingsObj.id}`)
+
+li.id = sightingsObj.id 
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```

Also, setting the `li.id` isn't necessary, since it's not part of this deliverable.